src/slicer/SlicerArrows.js

```diff
@@ -0,0 +1,187 @@
+        const oppositeSide = this.selectedArrow.properties.oppositeSide.getValue();
+        const oppositeArrow = this.arrows[oppositeSide];
+        oppositePosition3d = oppositeArrow.position.getValue(this.julianDate);
+      } else if (this.selectedArrow.properties.oppositePosition) {
+        oppositePosition3d = this.selectedArrow.properties.oppositePosition.getValue();
+      } else {
+        throw new Error('Move axis can\'t be created. Second position missing');
+      }
+
+      const arrowPosition3d = this.selectedArrow.position.getValue(this.julianDate);
+      const arrowPosition2d = scene.cartesianToCanvasCoordinates(arrowPosition3d);
+      const oppositePosition2d = scene.cartesianToCanvasCoordinates(oppositePosition3d);
+
+      // get pixel size for calculation move distance in meters
+      const boundingSphere = new BoundingSphere(arrowPosition3d);
+      const pixelSize = scene.camera.getPixelSize(boundingSphere, scene.drawingBufferWidth, scene.drawingBufferHeight);
+
+      // calculate scalar of mouse move
+      const axisVector2d = Cartesian2.subtract(oppositePosition2d, arrowPosition2d, new Cartesian2());
+      const mouseMoveVector = Cartesian2.subtract(movement.endPosition, arrowPosition2d, new Cartesian2());
+      const scalar2d = Cartesian2.dot(mouseMoveVector, axisVector2d) / Cartesian2.dot(axisVector2d, axisVector2d);
+
+      // calculate distance in meters
+      const objectMoveVector2d = Cartesian2.multiplyByScalar(axisVector2d, scalar2d, new Cartesian2());
+      const newArrowPosition2d = Cartesian2.add(arrowPosition2d, objectMoveVector2d, new Cartesian2());
+      const distance = Cartesian2.distance(newArrowPosition2d, arrowPosition2d) * pixelSize;
+
+      // calculate Cartesian3 position of arrow
+      const scalarDirection = (1 / scalar2d) * Math.abs(scalar2d);
+      const scalar3d = distance / Cartesian3.distance(arrowPosition3d, oppositePosition3d) * scalarDirection;
+      const axisVector3d = Cartesian3.subtract(oppositePosition3d, arrowPosition3d, new Cartesian3());
+      const objectMoveVector3d = Cartesian3.multiplyByScalar(axisVector3d, scalar3d, new Cartesian3());
+      const newArrowPosition3d = Cartesian3.add(arrowPosition3d, objectMoveVector3d, new Cartesian3());
+
+      // directly update arrow position if position callback not provided
+      if (!this.positionUpdateCallback) {
+        this.selectedArrow.position = newArrowPosition3d;
+      }
+      if (this.moveCallback) {
+        // calculate move amount (distance with direction)
+        const moveAmount = distance * getDirectionFromPoints(arrowPosition3d, newArrowPosition3d);
+
+        this.moveCallback(side, moveAmount, objectMoveVector3d);
+      }
+
+    } else {
+      this.highlightArrow(movement.endPosition);
+    }
+    this.viewer.scene.requestRender();
+  }
+
+  createMoveArrows() {
+    const arrowEntityTemplate = {
+      model: {
+        minimumPixelSize: 64,
```

Comment:
This should be configurable in the options. Move these default values at the top of the file.